index.html

```diff
@@ -10,9 +10,21 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
+<section>
```

Comment:
You should be able to link these to your img folder in your project.  They have the same images in them.